src/api/faq/faq.controller.js

```diff
@@ -43,4 +43,42 @@ async function getFAQ(req, res) {
   }
 }
 
-module.exports = { getFAQ };
+async function addFAQ(req, res) {
+  try {
+    const { question, answer } = req.body;
+    if (!question || !answer) throw new ReferenceError('Make sure your request includes a question and answer.');
+    await new FAQ({ question, answer }).save();
+    return res.json({ message: 'FAQ added successfully to DB.' });
+  } catch (error) {
+    const {
+      message,
+    } = error;
+    const status = error instanceof ReferenceError ? 404 : 500;
```

Comment:
Error code 400 would fit better, since it is an invalid/bad request.
`const status = error instanceof ReferenceError ? 400 : 500;`